config/webpack/webpack.config.js

```diff
@@ -218,15 +221,27 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
     resolveModules.unshift(path.resolve(rootPath, 'aggregated-translations'));
   }
 
-  const themeOverride = process.env.THEME; // Flexes root theme for theme visual regression testing.
-  const themeFile = ThemeAggregator.aggregate(themeOverride);
+  const defaultTheme = process.env.THEME; // Flexes root theme for theme visual regression testing.
+  const themeConfig = defaultTheme ? { theme: defaultTheme } : null;
+  let themeFile;
+  if (!disableAggregateThemes) {
+    // Set the default theme and disable scoped theme aggregation.
+    themeFile = ThemeAggregator.aggregate(
+      null, // Set the default theme via the config.
+      {
+        config: themeConfig,
+        aggregateDefaultThemeAsScopedTheme: true,
+      },
+    );
+  }
 
   const options = {
     rootPath,
     resolveModules,
     staticOptions,
     aggregatedLocales,
     themeFile,
+    defaultTheme,
```

Comment:
Why not pass the themeConfig object created on line 225 instead of re-creating it in the [webpack config](https://github.com/cerner/terra-toolkit/pull/390/files#diff-4acfd133078383adaf48ec081882c927R33)?

Reply:
Passivity? 🤷‍♂ I think the change makes sense.